### DISCUSSION TITLE
How to ignore lines in Laravel Pint?

### DISCUSSION THREAD

**Discussion by martio** [laravel/framework#44554] [Q&A]
How to ignore lines in Laravel Pint? Like `// @PhpCsFixer::ignore{"lowercase_static_reference"}`...

**Comment by asamofal** [comment]
Any updates here? Is there a way to ignore some [parts of a file](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file)?

For example, when using a custom validation rule, it requires calling it without parentheses:
<img width="924" alt="image" src="https://github.com/laravel/framework/assets/2729547/7562e6d9-42fb-4d77-b4f5-bb668cd0f4a5">
 
As you may see with phpcs it's possible to ignore the line, what to do with Pint (if I don't want to disable this rule globally)?

**Reply by martinbean** [reply]
> when using a custom validation rule, it requires calling it without parentheses

@asamofal Why? It’s just a class:

```php
'phone' => ['required', new Phone()],
```

**Comment by asamofal** [comment]
> @asamofal Why? It’s just a class:
> 
> ```
> 'phone' => ['required', new Phone()],
> ```

@martinbean Yes, right you are. I knew that a custom validation rule should be just an instance of a class but it was not working for me (with parentheses) for an unknown reason. But I tried again and now it works, so I had something else that time.

But anyway, even if my example was not correct, the idea of ignoring a single line/method/class sounds very natural, isn't?
